public/docs/_examples/displaying-data/dart/lib/app_component_1.dart

```diff
@@ -0,0 +1,15 @@
+// #docregion
+library displaying_data.app_component_1;
+
+import 'package:angular2/angular2.dart';
+
+@Component(
+    selector: 'my-app',
+    template: '''
+<h1>{{title}}</h1>
+<h2>My favorite hero is: {{myHero}}</h2>''')
+class AppComponent {
+  String title = 'Tour of Heroes';
+  String myHero = 'Windstorm';
+}
+// #enddocregion
```

Comment:
I never bothered with `#enddocregion` since the EOF is implicitly the end. Does the original have this comment? (Just curious; I don't think it matters.)